CoreScriptsRoot/Modules/Settings/Utility.lua

```diff
@@ -1059,13 +1052,21 @@ local function CreateSelector(selectionStringTable, startPosition)
 				selectionLabel.Position = tweenPos
 				selectionLabel.Visible = true
 				PropertyTweener(selectionLabel, "TextTransparency", 1, 0, TweenTime * 1.1, EaseOutQuad)
-				selectionLabel:TweenPosition(UDim2.new(0,leftButton.Size.X.Offset,0,0), Enum.EasingDirection.In, Enum.EasingStyle.Quad, TweenTime, true)
+				if selectionLabel:IsDescendantOf(game) then
+					selectionLabel:TweenPosition(UDim2.new(0,leftButton.Size.X.Offset,0,0), Enum.EasingDirection.In, Enum.EasingStyle.Quad, TweenTime, true)
```

Comment:
one last nitpicky thing, can we define the UDim2 before the if/else? Don't want someone to accidentally only update one of these in the future